srcpkgs/ipp-usb/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'ipp-usb'
+pkgname=ipp-usb
+version=0.9.17
+revision=1
+build_style=go
+go_import_path="github.com/OpenPrinting/ipp-usb"
+conf_files="/etc/ipp-usb/ipp-usb.conf"
+hostmakedepends="pkg-config"
+makedepends="avahi-libs-devel libusb-devel"
+depends="avahi>=0.8"
```

Comment:
I don't think you need to specify that version, it's been in void for a long while now. Just depending on `avahi` is correct.